cfn/ESDK-Hierarchy-CI.yaml

```
@@ -103,6 +107,42 @@ Resources:
           Projection:
             ProjectionType: "ALL"
 
+  # Static Key Store table for storing static branch keys
+  KeyStoreTestStaticTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      AttributeDefinitions:
+        - AttributeName: "branch-key-id"
+          AttributeType: "S"
+        - AttributeName: "type"
+          AttributeType: "S"
+      BillingMode: PAY_PER_REQUEST
+      KeySchema:
+        - AttributeName: "branch-key-id"
+          KeyType: "HASH"
+        - AttributeName: "type"
+          KeyType: "RANGE"
+      TableName: !Ref KeyStoreStaticTable
+
+  StaticKeyStoreTableReadOnly:
+    Type: "AWS::IAM::ManagedPolicy"
+    Properties:
+      Description: "Allow Read-Only access to Static Key Store Table"
+      ManagedPolicyName: !Sub "${ProjectName}-DDB-ReadOnly-${AWS::Region}"
```

Suggestion: Put the table name in the policy.
```suggestion
      ManagedPolicyName: !Sub "${ProjectName}-DDB-${KeyStoreStaticTable}-ReadOnly-${AWS::Region}"
```